types/unsplash-js/index.d.ts

```diff
@@ -7,6 +7,19 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
+export type PhotoOrderBy = 'latest' | 'oldest';
+export type Orientation = 'portrait' | 'landscape' | 'squareish';
+export type ContentSafety = 'high' | 'low';
+export type CollectionIds = ReadonlyArray<number> | ReadonlyArray<string>;
```

Comment:
If you want to allow a mixture of string and number in the same array, you could make this `ReadonlyArray<string | number>`.

Reply:
Ah that's actually what I wanted to do, thanks!